docker-compose.yml

```diff
@@ -34,14 +34,15 @@ services:
       - SFTP_HOST=dev-print-file-service-sftp
       - SFTP_PORT=22
       - SFTP_USERNAME=centos
-      - SFTP_KEY_FILENAME=/app/dummy_keys/dummy_rsa
+      - SFTP_KEY_FILENAME=/home/printfile/dummy_keys/dummy_rsa
       - SFTP_PASSPHRASE=secret
       - ENVIRONMENT=DEV
-      - QUARANTINED_FILES_DIRECTORY=/working_files/quarantined_files
-      - PARTIAL_FILES_DIRECTORY=/working_files/partial_files
-      - ENCRYPTED_FILES_DIRECTORY=/working_files/encrypted_files
+      - QUARANTINED_FILES_DIRECTORY=/home/printfile/working_files/quarantined_files
```

Comment:
we would have to consider the migration of in-flight files being processed during a release - but as this wont go in till after rehearsal now it's not a problem